style-guide/how-to-article-guidelines.rst

```diff
@@ -0,0 +1,500 @@
+
+.. _gerunds-and-participles:
+
+Clarify gerunds and participles
+-------------------------------
+
+Participles are verbs that end in *-ed* or *-ing* and act as modifiers.
+Gerunds are verbs that end in *-ing* and act as nouns. Both types of words are
+useful and acceptable, but confusion can arise if they are not placed
+precisely in a sentence. For example, the word *meeting* can be a gerund or a
+modifier (or even a noun) depending on its placement in a sentence. Clarify
+gerunds and participles as necessary.
+
+For more information and examples, see the `Clarify gerunds and participles <https://developer.rackspace.com/docs/style-guide/writing/clarify-gerunds-participles>`__ section of this style guide.
+
+.. _code-examples-guidelines:
+
+Write clear and consistent code examples
+----------------------------------------
+
+Observe the following guidelines when creating blocks of code as input or
+output examples:
+
+- Do not use screenshots to show code examples. Format them as blocks of code,
+  in monospace, by using the appropriate markup in your authoring tool.
+
+- When showing input, include a command prompt (such as $).
+
+- As often as necessary, show input and output in separate blocks and provide
+  explanations for each. For example, if the input contains arguments or
+  parameters, explain those. If the user should expect something specific in
+  the output, or you want to show only part of lengthy output, provide an
+  explanation. Provide your users the information that they need, and separate
+  the input and output when it makes sense.
+
+- When the command is simple, and there is nothing specific to say about the
+  output, you can show the input and output in the same code block, as the
+  user would actually see the code in their own terminal. The inclusion of the
+  command prompt will differentiate the input from the output.
+
+- Ensure that any placeholder text in code is obvious. If the authoring tool
+  allows it, apply italics to placeholders; if not, enclose them in angle
+  brackets. Show all placeholder text in camelCase.
+
+- Follow the conventions of the programming language used and preserve the
+  capitalization that the author of the code used.
+
+- For readability, you can break up long lines of input into blocks by ending
+  each line with a backslash.
+
+- If the input includes a list of arguments or parameters, show the important
+  or relevant ones first, and group related ones. If no other order makes
+  sense, use alphabetical order. If you explain the arguments or parameters in
+  text, show them in the same order that they appear in the code block.
+
+The following example illustrates many of these guidelines. For more examples,
+see the `Code examples <https://developer.rackspace.com/docs/style-guide/style/code-examples/>`__ section of this style guide.
+
+**Example: Create a VM running a Docker host**
+
+1. Show all the available virtual machines (VMs) that are running Docker.
```

Comment:
Did you want to add "...of the available..." and "...by running the following command:" here?

Reply:
Fixed